### PR TITLE
firewall: Improve the comments (documentation)

### PR DESCRIPTION
**Pull request by primeos**
###### Motivation for this change

Mainly some improvements to the comments, refactoring of the "documentation" and minor fixes (code and typos):

Optional: Some questions about the file formatting:
- Are there some official conventions we use for nixpkgs?
- This file uses two spaces between sentences. Why? (GNU/Emacs/19th Century (American) typist/typewriter convention)
- This file seems to wrap lines at about 72 characters (instead of e.g. 80) - is that correct?.

I'll make an issue (see #21974) to collect all my technical and controversial changes before opening PRs for them.

### CODE REVIEW

**Comment by domenkozar** [comment]
Experience so far shows that PRs are merged faster if fragmented :)

**Review by fpletz — APPROVED** [review]

**Comment by Mic92** [comment]
@domenkozar this is also empirically proven https://blog.jessfraz.com/post/analyzing-github-pull-request-data-with-big-query/

File: nixos/modules/services/networking/firewall.nix [hunk 1]
```diff
@@ -1,20 +1,31 @@
 /* This module enables a simple firewall.
 
-   The firewall can be customised in arbitrary ways by setting
+   The firewall can be customized in arbitrary ways by setting
```
Comment (edolstra) [inline nixos/modules/services/networking/firewall.nix]: No reason to change British to US spelling...

Comment (primeos) [inline nixos/modules/services/networking/firewall.nix]: Sorry, my bad, I wasn't aware of the British spelling...

File: nixos/modules/services/networking/firewall.nix [hunk 2]
```diff
@@ -4,17 +4,29 @@
...
      the packet.)  If you want additional logging, or want to accept
      certain packets anyway, you can insert rules at the start of
-     these chain.
+     this chain.
 
-   - ‘nixos-fw-accept’ is called for accepted packets.  If you want
-     additional logging, or want to reject certain packets anyway, you
-     can insert rules at the start of this chain.
+   - ‘nixos-fw-rpfilter’ is used as the main chain in the raw table,
+     called from the build-in ‘PREROUTING’ chain.  If the kernel supports it
```
Comment (globin) [inline nixos/modules/services/networking/firewall.nix]: "build-in" -> "built-in"

Comment (primeos) [inline nixos/modules/services/networking/firewall.nix]: Fixed, thanks :smile:

**Pull request merged**